schema.go

```diff
@@ -2193,7 +2193,39 @@ var Schema = schema.NewTypedScopeSchema[*Config](
 	// region PersistentVolumeClaimVolumeSource
 	schema.NewStructMappedObjectSchema[v1.PersistentVolumeClaimVolumeSource](
 		"PersistentVolumeClaimVolumeSource",
-		map[string]*schema.PropertySchema{},
+		map[string]*schema.PropertySchema{
+			"claimName": schema.NewPropertySchema(
+				dnsSubdomainName,
+				schema.NewDisplayValue(
+					schema.PointerTo("claimName"),
+					schema.PointerTo(
+						"claimName is the name of a PersistentVolumeClaim in the same namespace "+
+							"as the pod using this volume.",
+					),
+					nil,
+				),
+				true,
+				nil,
+				nil,
+				nil,
+				nil,
+				nil,
+			),
+			"readOnly": schema.NewPropertySchema(
+				schema.NewBoolSchema(),
+				schema.NewDisplayValue(
+					schema.PointerTo("readOnly"),
+					schema.PointerTo("readOnly Will force the ReadOnly setting in VolumeMounts."),
```

Comment:
You don't want to make the more direct and obvious claim that this property marks the volume read-only?

Reply:
It's just copy-pasted from the Golang documentation. Do you want us to use a different description?

Reply:
I generally really dislike this sort of empty self-referential documentation. In this case, "readonly" has obvious implications, but "readonly will force [the] readonly setting" is pretty pointlessly reductive. So, "whatever". If you don't feel like tweaking it, consider it approved.

Reply:
If this was any more complex, I would add additional documentation. But in this case, it's literally read only. That's as self explanatory as we can get. But this documentation clarifies that it isn't the direct setting, and that it only forces a setting elsewhere, so I will keep it as-is.